### ISSUE TITLE
Scope and ancestors question

### ISSUE THREAD

**Issue by williamoliveira** [lazychaser/laravel-nestedset#221]
Is it needed to use `scoped` everytime I need to access `ancestors` ?

```
>>> \App\Models\Category::scoped(['office_id' => 1])->find(5)
=> App\Models\Category {#944
     id: 5,
     name: "et",
     _lft: 8,
     _rgt: 9,
     parent_id: 1,
     office_id: 1,
     ancestors: Kalnoy\Nestedset\Collection {#963
       all: [
         App\Models\Category {#956
           id: 1,
           name: "Root",
           _lft: 1,
           _rgt: 12,
           parent_id: null,
           office_id: 1,
           ancestors: Kalnoy\Nestedset\Collection {#965
             all: [],
           },
         },
       ],
     },
   }
>>> \App\Models\Category::find(5)
=> App\Models\Category {#947
     id: 5,
     name: "et",
     _lft: 8,
     _rgt: 9,
     parent_id: 1,
     office_id: 1,
     ancestors: Kalnoy\Nestedset\Collection {#976
       all: [],
     },
   }
```
My model:
```
class Category extends Model
{
    use NodeTrait;

    protected $table = 'categories';

    public $timestamps = false;

    protected $fillable = [
        'name',
        'office_id',
    ];

    protected $with = [
        'ancestors',
    ];

    protected function getScopeAttributes()
    {
        return ['office_id'];
    }

    public function office()
    {
        return $this->belongsTo(Office::class);
    }
}
```

**Comment by lazychaser** [comment]
Eager loading with scoping is fixed in 4.2.6

**Comment by williamoliveira** [comment]
Thank you
